opensrp-anc/src/test/java/org/smartregister/anc/library/task/AttentionsFlagsTaskTest.java

```diff
@@ -19,30 +17,28 @@
 import java.util.Map;
 
 @RunWith(RobolectricTestRunner.class)
-public class AttentionsFlagsTaskTest {
-    private static BaseHomeRegisterActivity baseHomeRegisterActivity;
+public class AttentionsFlagsTaskTest extends BaseUnitTest {
+    AttentionFlagsTask attentionFlagMock;
```

Comment:
I thought we removed Aync tasks? Why do we still have this here?

Reply:
This is a test for class AttentionFlagsTask. The class name remained but doesn't extend async tasks as before.

Reply:
ooh, I see. thanks.